src/coreclr/tools/aot/ILCompiler.Compiler/Compiler/Logging/DocumentationSignatureParser.cs

```diff
@@ -631,10 +628,9 @@ private static void GetMatchingMethods(string id, ref int index, TypeDesc type,
             index = endIndex;
         }
 
-#if false
-        static void GetMatchingProperties(string id, ref int index, TypeDesc type, string memberName, List<TypeSystemEntity> results, bool acceptName = false)
+        private static void GetMatchingProperties(string id, ref int index, TypeDesc typeDesc, string memberName, List<TypeSystemEntity> results, bool acceptName = false)
```

Comment:
I think this fixes #75994.

Reply:
It all actually started with that bug, but as I was trying to fix it I ran all kinds of things. This PR is the first result. Then when I tried to port over the linker tests for that bug I ran into the problem of comparing Cecil/TypeSystem entities precisely, which I haven't really solved yet.

So yes - this probably fixes the bug in the product, but I still need to port over the tests for it.